test/controllers/user.js

```diff
@@ -181,110 +207,152 @@ describe('User API calls', function () {
         .set('Authorization', `Bearer ${user.token}`)
         .expect(200);
 
-      expect(res.body.email).to.equal(update.email);
+      //expect(res.body.email).to.equal(update.email);
       expect(res.body.name).to.equal(update.name);
       expect(res.body.birthdate).to.contain(update.birthdate);
       expect(res.body.locale).to.equal(update.locale);
 
       expect(res.body.role).to.equal('user');
       expect(res.body.password).not.to.equal(update.password);
+      await helper.deleteMochaUserById(user.userId); //when email update delete with Id
```

Comment:
I think this is redundant, because we already have a `afterEach` block that does this.